### ISSUE TITLE
Invalid command line execution throws TypeError: Cannot read property 'group' of undefined

### ISSUE THREAD

**Issue by erdii** [mpneuried/rsmq-cli#3]
when rsmq-cli is executed with invalid arguments like `rsmq -g rmon rc "abc"` or `rsmq create "myqueue"` it throws the following error:
```
/usr/local/lib/node_modules/rsmq-cli/lib/_rsmq.js:52
      _cnf = cnf.read(options.group);
                             ^

TypeError: Cannot read property 'group' of undefined
    at new RSMQCli (/usr/local/lib/node_modules/rsmq-cli/lib/_rsmq.js:52:30)
    at module.exports (/usr/local/lib/node_modules/rsmq-cli/lib/_rsmq.js:311:12)
    at Command.commands.action (/usr/local/lib/node_modules/rsmq-cli/lib/main.js:57:16)
    at Command.listener (/usr/local/lib/node_modules/rsmq-cli/node_modules/commander/index.js:301:8)
    at emitTwo (events.js:87:13)
    at Command.emit (events.js:172:7)
    at Command.parseArgs (/usr/local/lib/node_modules/rsmq-cli/node_modules/commander/index.js:615:12)
    at Command.parse (/usr/local/lib/node_modules/rsmq-cli/node_modules/commander/index.js:458:21)
    at Object.<anonymous> (/usr/local/lib/node_modules/rsmq-cli/lib/main.js:182:7)
    at Object.<anonymous> (/usr/local/lib/node_modules/rsmq-cli/lib/main.js:184:4)
```

**Comment by joaosa** [comment]
I'm also experiencing this issue

**Comment by mpneuried** [comment]
Fixed error handling with 2b90c268a17659310887a0575123b7cfd8ac61a8 in version 0.1.3
